### PR TITLE
10.9.0 Release

### PR DESCRIPTION
**Pull request by chrisbreiding**
- [x] https://github.com/cypress-io/cypress-documentation/pull/4752
- [x] https://github.com/cypress-io/cypress-documentation/pull/4681
- [x] https://github.com/cypress-io/cypress-documentation/pull/4746
- [x] https://github.com/cypress-io/cypress-documentation/pull/4753
- [x] https://github.com/cypress-io/cypress-documentation/pull/4730

### CODE REVIEW

File: content/api/commands/origin.md [hunk 1]
```diff
@@ -473,49 +505,115 @@ of
...
+  // ... code to run before each test ...
+})
+```
+
+`cypress/e2e/spec.cy.js`:
+
+```js
+it('tests somesite.com', () => {
+  cy.origin('somesite.com', () => {
+    // makes custom commands available to all subsequent
```
Comment (BlueWinds) [inline content/api/commands/origin.md:561]: Is this all subsequent calls within this test, within this spec, or within this cypress run?

File: content/api/commands/origin.md [hunk 2]
```diff
@@ -233,11 +234,42 @@ and the protocol defaults to `https`. When `cy.visit()` is called with the path
...
+cy.origin('www.acme.com', () => {
+  cy.get('h1').contains('About our Founder, Marvin Acme')
+})
+```
+
+Here the cross-origin page is visited prior to the `cy.origin` block, but any
+interactions with the window are performed within the block which can
+communicate with the cross-origin page
+
+#### Incorrect usage
```
Comment (BlueWinds) [inline content/api/commands/origin.md:254]:
```suggestion
#### **<Icon name="exclamation-triangle" color="red"></Icon> Incorrect Usage**
```

Should this have the red exclamation triangle?

**Review by mjhenkes — APPROVED** [review]

**Review by admah — APPROVED** [review]

**Review by AtofStryker — APPROVED** [review]

File: content/api/commands/origin.md (hunk 1 above)
Comment (chrisbreiding) [reply content/api/commands/origin.md]: Addressed in https://github.com/cypress-io/cypress-documentation/pull/4747/commits/59809e4535f7b28141413884c0c21767bffeed94

File: content/api/commands/origin.md (hunk 2 above)
Comment (chrisbreiding) [reply content/api/commands/origin.md]: LGTM:

Screen Shot 2022-09-27 at 12 21 33 PM

Addressed in https://github.com/cypress-io/cypress-documentation/pull/4747/commits/59809e4535f7b28141413884c0c21767bffeed94

File: content/api/cypress-api/require.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+const callback = () => {
+  const _ = Cypress.require('lodash')
+})
+
+cy.origin('somesite.com', callback)
+```
+
+## Examples
+
+See [`cy.origin()` Dependencies / Sharing Code]() for example usages.
```
Comment (elylucas) [inline content/api/cypress-api/require.md:56]: This is currently missing the link

**Review by elylucas — CHANGES_REQUESTED** [review]

File: content/api/cypress-api/require.md (hunk 3 above)
Comment (elylucas) [inline content/api/cypress-api/require.md:56]:
```suggestion
See [`cy.origin()` Dependencies / Sharing Code](/api/commands/origin#Serialization) for example usages.
```

Comment (chrisbreiding) [reply content/api/cypress-api/require.md]: Good catch! Fixed in https://github.com/cypress-io/cypress-documentation/pull/4747/commits/5653543ab352b0a62ce3a66039c5da3209c29123

**Pull request merged**